### PR TITLE
Added mantine, improved the bridge, gas, and token standards pages

### PR DESCRIPTION
**Pull request by codebycarson**
- Added mantine UI and matched the theme to "next-themes"
- Improved the developer intro, chains, gas, token standards, and smart contracts sections
- Moved interoperability to it's own section and split the pointer contracts into two sections
- Small fixes to governance and staking pages

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import '../styles/globals.css';
+import '../styles/custom.css';
+import '@rainbow-me/rainbowkit/styles.css';
+import { ThemeProvider } from 'next-themes';
+import '@mantine/core/styles.css';
+import { createTheme, MantineProvider } from '@mantine/core';
+
+const theme = createTheme({
```
Comment (besated) [inline pages/_app.tsx]: should a theme be specified here for colors?

Comment (codebycarson) [reply pages/_app.tsx]: Yeah, I was just getting it in so devrel could use it, but I suppose I can add a theme.

File: pages/dev-ecosystem-providers/bridges.mdx
Comment (cordt-sei) [inline pages/dev-ecosystem-providers/bridges.mdx:1]: Should mention [CCTP](https://cctp.money/?to=sei&from=noble) being the canonical / native USDC.

File: pages/_app.tsx (hunk 1 above)
Comment (cordt-sei) [reply pages/_app.tsx]: I can add a theme here. Dark and light with Sei colors?

File: pages/dev-gas.mdx [hunk 2]
```diff
@@ -1,41 +1,46 @@
...
-## Optimizing Gas Prices for Smart Contracts
-
-Optimizing gas prices involves efficient smart contract coding practices:
-- Minimize storage operations.
-- Use fixed-size data structures where possible.
-- Avoid unnecessary computations.
+#### Current values
+- Mainnet (pacific-1): `0.025usei`
+- Testnet (atlantic-2): `0.01usei`
+- Devnet (arctic-1): `0.005usei`
```
Comment (besated) [inline pages/dev-gas.mdx]: i think this should be:
- Mainnet (pacific-1): `0.02usei`
- Testnet (atlantic-2): `0.08usei`
- Devnet (arctic-1): `0.02usei`

File: pages/dev-ecosystem-providers/bridges.mdx
Comment (codebycarson) [reply pages/dev-ecosystem-providers/bridges.mdx:1]: We are going to pull these from the same API the main sei page does and keep that as the source of truth.

**Review by besated — APPROVED** [review]

**Review by besated — APPROVED** [review]

**Pull request merged**